erreser.go

```
@@ -0,0 +1,658 @@
+	db := database.DbConnect(dbParams)
+	var feed models.Feed
+	db.Where(&models.Feed{Title: feedTitle}).First(&feed)
+	log.Println("Checking feed: ", feed.Title)
+	fp := gofeed.NewParser()
+	feedData, err := fp.ParseURL(feed.Feed)
+	if err != nil {
+		log.Println("Error parsing feed: ", err)
+		return
+	}
+	if len(feedData.Items) > 200 {
+		updateItems(db, feedData.Items[:200], &feed)
```

golangci-lint (1.62.2)

Error return value is not checked

(errcheck)